it/pom.xml

```diff
@@ -26,6 +26,7 @@
     <beam.version>2.38.0</beam.version>
     <cloud-bom.version>24.2.0</cloud-bom.version>
     <cloud-storage.version>2.2.2</cloud-storage.version>
+    <google-cloud-bigquery.version>1.108.0</google-cloud-bigquery.version>
```

Comment:
Just curious - do we need to specify a version? isn't it declared in the `libraries-bom`?